src/components/Contact.tsx

```diff
@@ -29,10 +36,13 @@ const Contact = () => {
           {t("contact.header")}
         </H1>
         <ContactInfo fontSize={[4, 4, 4, 5]}>
-          {t("contact.contact-info")}
+          FOR INQUIRIES PLEASE CONTACT{" "}
```

Comment:
do we not need translation here?

Reply:
if we dont, have you remembered to remove the relevant text from the translation files